OpenTabletDriver/InputDevice.cs

```diff
@@ -67,6 +78,8 @@ protected override bool Initialize()
                 {
                     ReportStream.SetFeature(report);
                     Log.Debug("Device", "Set device feature: " + BitConverter.ToString(report));
+                    if (FeatureInitDelayMs != 0)
+                        Thread.Sleep((int)FeatureInitDelayMs);
```

Comment:
```suggestion
                    if (FeatureInitDelayMs != 0)
                        Thread.Sleep((int)FeatureInitDelayMs);
                    ReportStream.SetFeature(report);
                    Log.Debug("Device", "Set device feature: " + BitConverter.ToString(report));
```
There should be a delay in between the GetString call and SetFeature.